docs/data/material/guides/interoperability/interoperability.md

```diff
@@ -614,14 +614,16 @@ If you are used to Tailwind CSS and want to use it together with the MUI compone
 If you use a different framework, or already have set up your project, follow these steps:
 
 1. Add Tailwind CSS to your project, following the instructions in https://tailwindcss.com/docs/installation.
-2. Remove Tailwind's `base` directive in favor of the `CssBaseline` component provided by `@mui/material`, as it plays nicer with the MUI components.
+2. Remove Tailwind's `base` styles in favor of the `CssBaseline` component provided by `@mui/material`, as it plays nicer with the MUI components. More details on tailwind base styles can be found [here](https://tailwindcss.com/docs/preflight#disabling-preflight)
 
-**index.css**
+**tailwind.config.js**
 
 ```diff
--@tailwind base;
- @tailwind components;
- @tailwind utilities;
+module.exports = {
++  corePlugins: {
++    preflight: false,
++  }
+}
```

Comment:
This is how git outputs its diff, plus keeping our prettier/esling code format config:

```suggestion
 module.exports = {
+  corePlugins: {
+    preflight: false,
+  },
 };
```